crates/native/Cargo.toml

```diff
@@ -17,6 +17,7 @@ flutter_rust_bridge = "1.23"
 lazy_static = "1.4"
 libwebrtc-sys = { path = "../libwebrtc-sys" }
 log = "0.4"
+md5 = "0.7.0"
```

Comment:
```toml
xxhash-rust = { version = "0.8", features = ["xxh3"] }
```
[xxHash](https://docs.rs/xxhash-rust) should be much more performant here, and suit well into `u32`/`u64`/`u128` types on your choice.